<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IDE</class>
 <widget class="QMainWindow" name="IDE">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IDE</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QTextEdit" name="textEdit_2"/>
    </item>
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
    <item row="1" column="0">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>922</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件（&amp;F)</string>
    </property>
    <addaction name="action_N"/>
    <addaction name="action_O"/>
    <addaction name="action_W"/>
    <addaction name="separator"/>
    <addaction name="action_S"/>
    <addaction name="action_M"/>
    <addaction name="separator"/>
    <addaction name="action_Q"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑（&amp;E）</string>
    </property>
    <addaction name="action_Z"/>
    <addaction name="action_X"/>
    <addaction name="separator"/>
    <addaction name="action_C"/>
    <addaction name="action_V"/>
    <addaction name="separator"/>
    <addaction name="action_F"/>
   </widget>
   <widget class="QMenu" name="menu_b">
    <property name="title">
     <string>构建（&amp;B）</string>
    </property>
    <addaction name="action_I"/>
    <addaction name="action_R"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_b"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_N">
   <property name="text">
    <string>新建（&amp;N）</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_O">
   <property name="text">
    <string>打开（&amp;O）</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_W">
   <property name="text">
    <string>关闭（&amp;W）</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="action_S">
   <property name="text">
    <string>保存（&amp;S）</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_M">
   <property name="text">
    <string>另存为（&amp;M）</string>
   </property>
   <property name="toolTip">
    <string>另存为（M）</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="action_Q">
   <property name="text">
    <string>退出（&amp;Q）</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Z">
   <property name="text">
    <string>撤销（&amp;Z）</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_X">
   <property name="text">
    <string>剪切（&amp;X）</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_C">
   <property name="text">
    <string>复制（&amp;C）</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_V">
   <property name="text">
    <string>粘贴（&amp;V）</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_F">
   <property name="text">
    <string>查找（&amp;F）</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_I">
   <property name="text">
    <string>编译（&amp;I）</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="action_R">
   <property name="text">
    <string>运行（&amp;R）</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
